react/components/LoginAsCustomer.js

```diff
@@ -43,17 +43,17 @@ export default class LoginAsCustomer extends Component {
 
     return (
       <div className="vtex-telemarketing__login">
-        <Popover renderHeader={this.handleHeaderRendering}>
-          <div className="bg-black-90 w-100 pa4">
+        <Popover arrowUpColor="bg-base--inverted" renderHeader={this.handleHeaderRendering}>
```

Comment:
In `arrowUpColor="bg-base--inverted"`, `arowUpCollor` really receives a CSS class?

Reply:
Yes, it receives the design token class for the specific scenario